webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityHeadSpace.vue

```diff
@@ -2,8 +2,8 @@
   <a
     :id="id"
     :href="url"
-    :class="!this.space.isMember && 'not-clickable-link hidden-space'"
-    class="text-none space-avatar primary--text activity-head-space-link">
+    :class="testvisibility && 'not-clickable-link hidden-space'"
+    class="text-none space-avatar activity-head-space-link">
```

Comment:
why did you remove primary--text ?

Reply:
I remove the class primary--text to make the hidden space name be with the grey color

Reply:
and If the link is clickable, does primary--text is used ?

Reply:
yes, primary--text is not used only if the space is hidden and not clickable